src/Cafe/OS/libs/nn_act/nn_act.cpp

```diff
@@ -125,6 +125,21 @@ namespace act
 			return getNNReturnCode(resultCode, actRequest);
 		}
 
+		sint64 GetUtcOffset()
+		{
+			return ((ppcCyclesSince2000 / ESPRESSO_CORE_CLOCK) - (ppcCyclesSince2000_UTC / ESPRESSO_CORE_CLOCK)) * 1'000'000;
+		}
+
+		sint32 GetUtcOffsetEx(uint64* pOutOffset, uint8 slotNo)
```

Comment:
```suggestion
		sint32 GetUtcOffsetEx(sint64be* pOutOffset, uint8 slotNo)
```
This is probably a signed int since the offset can be negative, but more importantly it should be big endian?

Reply:
That's right, will fix!